src/client.ts

```diff
@@ -37,6 +37,9 @@ type ClientOptions = {
 export class Client {
   readonly storage: Storage;
   constructor(opts?: ClientOptions) {
+    const options: StorageOptions = {
+      userAgent: 'github-actions-upload-cloud-storage/0.2.0',
```

Comment:
This looks good but should we be automating it as part of the release-please process for future releases?

Reply:
I'll update it to grab the version from the package.json, which should be updated every release by release please

Reply:
Your process may  be better because I'm hitting the fact that package.json is not in the rootDir.

Reply:
Yeah I think release-please process maybe better because when the action is invoked from dist, it may not have access to package.json